### ISSUE TITLE
"unexpected directory layout" during go get

### ISSUE THREAD

**Issue by pwaller** [pwaller/docker-show-context#4]
Go 1.10.1, darwin/amd64.

```
$ go get -v -u github.com/pwaller/docker-show-context
github.com/pwaller/docker-show-context (download)
github.com/docker/docker (download)
unexpected directory layout:
	import path: github.com/docker/docker/builder/dockerignore
	root: /Users/pwaller/go/src
	dir: /Users/pwaller/go/src/github.com/pwaller/docker-show-context/vendor/github.com/docker/docker/builder/dockerignore
	expand root: /Users/pwaller/go/src
	expand dir: /Users/pwaller/go/src/github.com/pwaller/docker-show-context/vendor/github.com/docker/docker/builder/dockerignore
	separator: /
```

Not sure what's going on here. Not much time to investigate. Help welcomed!

**Comment by ryboe** [comment]
[Help has arrived.](https://github.com/pwaller/docker-show-context/pull/5) 😀

**Comment by pwaller** [comment]
Resolved in #6.
